src/game/hud.cpp

```diff
@@ -1415,7 +1415,7 @@ namespace hud
 
     bool showname()
     {
-        if(game::focus != game::player1)
+        if(game::focus == &game::player1)
```

Comment:
This clearly looks wrong. Why is that suddenly an equals comparison?

Reply:
guess a hasty replace I didn't noticed in my own code re-read.